lib/chitter_model.rb

```diff
@@ -0,0 +1,36 @@
+require 'pg'
+
+class Chitter
+
+  def self.get_peeps
+
+    if ENV['ENVIRONMENT'] == 'test'
+      con = PG.connect :dbname => 'Chitter_test', :user => 'evelynblazquez'
```

Comment:
Not sure if the :user part is necessary. Makes it difficult to run on a machine that's not yours